README.md

```diff
@@ -139,76 +169,136 @@ Once you've done that, restart the IDE.
 
 * [Access swagger api](http://localhost:8080/swagger-ui/index.html) and
   corresponding [openAPI docs here](http://localhost:8080/api-docs)
-    
+
 ## Quality Checks
 
 ### Jacoco
 
-* Generate Test reports and open [coverage report](build/reports/jacoco/test/html/index.html) 
+* Generate Test reports and open [coverage report](build/reports/jacoco/test/html/index.html)
+
 ```shell
 ./gradlew test jacocoTestReport
 ```
 
 * Check coverage minimum of 70%
+
 ```shell
 ./gradlew clean test jacocoTestCoverageVerification
 ```
 
-
 ### PMD
 
 * Run pmd for src
+
```

Comment:
Do you think worth adding a link to PMD documentation? As multiple volunteers asked about what is it?